appmonitoring/ts/src/server.ts

```diff
@@ -18,25 +18,27 @@ if ("secrets-manager".localeCompare(containerMode) === 0) {
     try {
         logger.info("Running in certificate manager mode...", operationId, null);
         logger.SendEvent("CertificateManagerModeRun", operationId, null, clusterArmId, clusterArmRegion);
-
         logger.addHeartbeatMetric(HeartbeatMetrics.CertificateOperationCount, 1);
-
         await CertificateManager.CreateWebhookAndCertificates(operationId, clusterArmId, clusterArmRegion);
-
         logger.info("Certificate manager mode is done", operationId, null);
         logger.SendEvent("CertificateManagerModeRunSuccess", operationId, null, clusterArmId, clusterArmRegion, true);
     } catch (error) {
         logger.addHeartbeatMetric(HeartbeatMetrics.CertificateOperationFaailedCount, 1);
-
         logger.error(`Certificate manager mode failed: ${JSON.stringify(error)}`, operationId, null);
         logger.SendEvent("CertificateManagerModeRunFailure", operationId, null, clusterArmId, clusterArmRegion, true, error);
-        
         throw error;
     }
-    
     process.exit();
-} 
-
+} else if ("secrets-housekeeper".localeCompare(containerMode) === 0) {
+    try {
```

Comment:
Add telemetry please, see _logger.addHeartbeatMetric_ and _logger.SendEvent_ calls above.